sar/README.md

```diff
@@ -223,16 +227,16 @@ which then requires one of `proj:wkt2` or `proj:projjson` to be specified.
 | --------------------------- | ------- | ----- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | sar:instrument_mode         | ✓ 1.6.4 | (✓)   | `ObservationMode`                                            | **REQUIRED for *Src*.**                                      |
 | sar:frequency_band          | ✓ 1.6.4 | (✓)   | `RadarBand`                                                  | **REQUIRED for *Src*.**                                      |
-| sar:center_frequency        | ✓ 1.6.4 | ✗     | `RadarCenterFrequency`                                       | **REQUIRED.** Convert to GHz if required.                    |
+| sar:center_frequency        | ✓ 1.6.4 | ✗     | `RadarCenterFrequency`                                       | **REQUIRED.** In hertz (Hz).                                 |
```

Comment:
We can't change this, the unit is defined in the SAR extension as GHz. Please revert (also in examples).